english/bake-cookies/cookies.md

```diff
@@ -6,6 +6,8 @@ want to spread the joy (and chocolate chip cookies, or peanut butter, or
 ginger).
 Probably you love cookies too, or at least you like the idea of them.
 Because cookies are awesome!!!
+If you don't have every ingredient you need for a recipe, try using alternatives, because you never know what you can get by experimenting!
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.